reporting-tricks.md

```diff
@@ -1,65 +1,49 @@
 # Analytics API Reporting Tips
 
-The new Analytics V2 API powers the Analysis Workspace project. Analysis Workspace includes a debugger that will let you view the different API requests it makes. This debugger is very helpful in learning how to craft report requests and exploring the full power of the new Analytics API.
+This document provides tips for using Analytics 2.0 APIs to view reports from Analysis Workspace and for using the **Try it out** feature in the Swagger interface. The Analytics 2.0 APIs power the Analysis Workspace project. Analysis Workspace includes a debugger that allows you to view the different API requests it makes. This debugger is helpful in learning how to craft report requests and in exploring Analytics 2.0 APIs.
```

Comment:
The goal of this document is to allow users to use see how to create API requests that match analysis workspace. They could even use analysis workspace as a tool to build the API requests that they are looking for once they are done. If they understand the power of this section it should make it much easier for them to find what they need and also reduce support requests. 
I often scan the first paragraph of the docs to see if I should read any more and I don't think that the this paragraph is clear enough on why I would want to read the document.

Reply:
Thanks for the clarification. I was thinking that the tip was to use the debugger to build the json body of the request, not that the tip was more than that. 

So, let me see if I understand this: by completing the steps, the user can understand the "full power" of the new APIs. What would be the best way to be more specific about the "full power?" That you can use the debugger for lots of requests in the future? I guess I need more clarification.

Here is what I now imagine is the goal of the doc, if I understand correctly:

1. Use AW debugger to see how to create API requests, including json bodies.
2. Get an experience using the Swagger interface with a json body post.
3. Identify what is needed for users to create an API request.

What would you modify or add in these goals? It might not be a bad idea for me to provide a numbered list of what the goals are at the beginning of the document.

Reply:
How is this?

This article provides an example of how to create API requests compatible with Analysis Workspace. Following this example, you can use Analysis Workspace to make your own additional requests after seeing how it works. 

This example includes the **Try it out** feature in the Swagger interface. It also includes using the Analysis Workspace debugger tool, which allows you to view the different API requests it makes. This debugger is helpful in understanding report requests and in exploring Analytics 2.0 APIs.

Or is the usage of "full power of the APIs" critical to convey the right idea?

Reply:
I like the new wording. You may want to take the "your own" out of the second sentence or streamline it in some other way since I stumbled over it a couple of times.

Reply:
ok. great. Thanks.